src/main/scala/jtag/JtagTap.scala

```diff
@@ -232,6 +227,8 @@ object JtagTapGenerator {
       }
     }
 
+    controllerInternal.io.dataChainIn := bypassChain.io.chainOut  // default
```

Comment:
why do we need this now?

Reply:
Same thing - FIRRTL reports "not fully initialized" -- this ensures there is a value.

Reply:
Same thing about "not fully initialized" due to not being assigned in some cases.  This ensures there is a value assigned in all cases.